File: uri_exercicios_em_Java/src/beginner/CoordinatesOfAPoint.java
package beginner;

import java.io.IOException;
import java.util.*;

public class CoordinatesOfAPoint {
    static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);

        float x = sc.nextFloat(), y = sc.nextFloat();


        if(x == 0 || y == 0){

            if(y != 0) System.out.println("Eixo Y");

            else if (x != 0) System.out.println("Eixo X");

            else System.out.println("Origem");

        }

        else{
            if(x > 0){
                if(y > 0) System.out.println("Q1");
                else System.out.println("Q4");
            }
            else{
                if(y > 0) System.out.println("Q2");
                else System.out.println("Q3");
            }
        }


    }

}

File: beecrowd_exercicios_em_python/begginner/coordinates_of_a_point.py
def quadrant(a, b):
    if a == 0 or b == 0:
        if b == 0 and a != 0:
            return "Eixo X"
        if a == 0 and b != 0:
            return "Eixo Y"

        return "Origem"

    else:
        if a > 0:
            if b > 0:
                return "Q1"

            return "Q4"

        if b < 0:
            return "Q3"

        return "Q2"


x, y = map(float, input().split())

print(quadrant(x, y))
